pkg/cache/clusterqueue.go

```diff
@@ -112,16 +122,16 @@ type queue struct {
 	admittedUsage resources.FlavorResourceQuantities
 }
 
-func newCohort(name string, size int) *cohort {
+func cohortFactory(name string) *cohort {
```

Comment:
factories usually return a function that creates something...
this is just creating a cohort, so the appropriate name is `newCohort` still.